about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="styles/about.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <nav class="navigation">
+                <div class="logo-wrap">
+                    <a href="index.html"><img src="images/logo 1.svg" alt="logo"></a>
+                </div>
+                <div class="navigation-header">
+                    <a href="">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="">Tech Attack</a>
+                    <a href="">Projects</a>
+                    <a href="">Contact</a>
+                    <a href=""><img src="images/Vector.svg" alt="logo-gthub"></a>
+                    <a href=""><img src="images/ant-design_twitter-circle-filled.svg" alt="logo-twitter"></a>
+                    <a href=""><img src="images/entypo-social_linkedin-with-circle.svg" alt="logo-linkedin"></a>
+                </div>
+            </nav>
+
+            <section class="container-about">
+                <div class="about-me">
+                    <h1>About Me</h1>
+                    <p>The Generator App is an online tool that helps you to export ready-made templates ready to work as your future website. It helps you to combine slides, panels and other components and export it as a set of static files: HTML/CSS/JS.</p>
+                </div>
+                    
+                <div class="work-experience">
+                    <h1>Work Experience</h1>
```

Comment:
Acho que faz mais sentido usar **`h2`** aqui
```suggestion
                    <h2>Work Experience</h2>
```

Reply:
Feito!